pkg/apb/types.go

```diff
@@ -50,6 +50,7 @@ type ParameterDescriptor struct {
 
 // Plan - Plan object describing an APB deployment plan and associated parameters
 type Plan struct {
+	ID             string                 `json:"id" yaml:"-"`
```

Comment:
Should these values be the same?

Reply:
I think that we don't want a value to be set from the Spec yaml for the ID field.

Reply:
👍 to @shawn-hurley we don't want it to get set form the spec yaml. But I want it output via JSON.